src/app/app.component.scss

```diff
@@ -1,10 +1,10 @@
 @import "../scss/variables";
 
 .footer {
-  position: fixed;
```

Comment:
@developerfred this fix works for the compose page, but degrades the inbox when there are not many messages:
![image](https://user-images.githubusercontent.com/2022378/68990885-7ade0d00-0850-11ea-815e-e06cc6505293.png)

It may be better to use `padding-bottom` on `#main-content`